AtReconstruction/AtFilter/AtSCACorrect.h

```diff
@@ -0,0 +1,58 @@
+#ifndef ATSCACORRECT_H
+#define ATSCACORRECT_H
+
+/**
+ * A filter to perform baseline subtraction using the average baseline and average phase effect stored as
+ * AtRawEvent objects in a separate file or files.
+ *
+ */
+
+#include "AtFilter.h"
+
+#include "TFile.h"
+#include "TString.h"
+
```

Comment:
Include `<memory> and "AtRawEvent.h"`
Forward declare: `class AtPad, struct AtPadReference`